This was unused in libvpx and removed later:

https://github.com/webmproject/libvpx/commit/763f8318de2cee41d748539cee5810fc0efcad68

This reverts https://bugzilla.gnome.org/show_bug.cgi?id=641399 until upstream
reverts it themselves.

--- ext/vpx/gstvpxdec.c
+++ ext/vpx/gstvpxdec.c
@@ -62,18 +62,6 @@ gst_vpx_dec_post_processing_flags_get_type (void)
     {C_FLAGS (VP8_DEBLOCK), "Deblock", "deblock"},
     {C_FLAGS (VP8_DEMACROBLOCK), "Demacroblock", "demacroblock"},
     {C_FLAGS (VP8_ADDNOISE), "Add noise", "addnoise"},
-    {C_FLAGS (VP8_DEBUG_TXT_FRAME_INFO),
-          "Print frame information",
-        "visualize-frame-info"},
-    {C_FLAGS (VP8_DEBUG_TXT_MBLK_MODES),
-          "Show macroblock mode selection overlaid on image",
-        "visualize-macroblock-modes"},
-    {C_FLAGS (VP8_DEBUG_TXT_DC_DIFF),
-          "Show dc diff for each macro block overlaid on image",
-        "visualize-dc-diff"},
-    {C_FLAGS (VP8_DEBUG_TXT_RATE_INFO),
-          "Print video rate info",
-        "visualize-rate-info"},
     {C_FLAGS (VP8_MFQE), "Multi-frame quality enhancement", "mfqe"},
     {0, NULL, NULL}
   };
